helm/qontract-reconcile/templates/template.yaml

```diff
@@ -141,15 +141,68 @@ objects:
           - name: fluentd-config
             mountPath: /fluentd/etc/
         {{- end }}
+        {{- if $integration.logs.cloudwatch }}
+        - name: syslog-cloudwatch-bridge
+          image: quay.io/app-sre/syslog-cloudwatch-bridge:latest
+          ports:
+            - containerPort: ${SYSLOG_PORT}
+              protocol: UDP
+              name: syslog-udp-port
+            - containerPort: ${SYSLOG_PORT}
+              protocol: TCP
+              name: syslog-tcp-port
+          env:
+            - name: PORT
+              value: ${SYSLOG_PORT}
+            - name: AWS_REGION
+              valueFrom:
+                secretKeyRef:
+                  name: ${CLOUDWATCH_SECRET}
+                  key: aws_region
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: ${CLOUDWATCH_SECRET}
+                  key: aws_access_key_id
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: ${CLOUDWATCH_SECRET}
+                  key: aws_secret_access_key
+            - name: LOG_GROUP_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: ${CLOUDWATCH_SECRET}
+                  key: log_group_name
+          resources:
+            requests:
+              memory: 30Mi
+              cpu: 15m
+            limits:
+              memory: 60Mi
+              cpu: 25m
```

Comment:
Are you sure this is enough? How about `100m`?

Reply:
I will change it